### PR TITLE
Update en.po to reword the tutorial

### PR DESCRIPTION
**Pull request by ender-titan1**
EDITOR_TUTORIAL_REMOVE_ORGANELLE_TEXT reworded to include the organelle menu.

_Closes #2118_

### CODE REVIEW

File: locale/en.po [hunk 1]
```diff
@@ -2592,7 +2592,10 @@ msgstr ""
 msgid "EDITOR_TUTORIAL_REMOVE_ORGANELLE_TEXT"
 msgstr ""
 "Removing organelles also costs MP as it is a mutation to your species. You "
-"can right click organelles to remove them.\n"
+"can right click organelles to bring up the organelle menu.\n"
+"In the organelle menu you can select the delete option to \n"
```
Comment (Kasterisk) [inline locale/en.po:2596]: Just need a bit of tweaking:
`...can right click organelles to bring up the organelle menu and selecting the delete option to remove them.`

**Review by Kasterisk — CHANGES_REQUESTED** [review]

File: locale/en.po (hunk 1 above)
Comment (ender-titan1) [reply locale/en.po]: Ok, tweaked the text as requested.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
This doesn't address my comment I left on the related issue: https://github.com/Revolutionary-Games/Thrive/issues/2118#issuecomment-785804547

**Comment by ender-titan1** [comment]
what should I change?
Make it shorter?

**Comment by hhyyrylainen** [comment]
Edit the editor tutorials scene to make the box bigger.

**Comment by ender-titan1** [comment]
in Godot, right?

**Comment by hhyyrylainen** [comment]
Yes.

File: locale/en.po [hunk 2]
```diff
@@ -2592,7 +2592,9 @@ msgstr ""
 msgid "EDITOR_TUTORIAL_REMOVE_ORGANELLE_TEXT"
 msgstr ""
 "Removing organelles also costs MP as it is a mutation to your species. You "
-"can right click organelles to remove them.\n"
+"can right click organelles to bring up the organelle menu "
+"and and selecting the delete option to remove them. \n"
```
Comment (RacerBG) [inline locale/en.po:2596]: You could remove one of the two "and's" here.

**Comment by ender-titan1** [comment]
Ok, resized the popup vertically.
But how do I commit that (I'm new to GitHub)?

**Comment by hhyyrylainen** [comment]
https://gist.github.com/Chaser324/ce0505fbed06b947d962
or you can use some GUI Git tool as well.

**Comment by ender-titan1** [comment]
Ok, I think it works.

**Review by Kasterisk — CHANGES_REQUESTED** [review]
The box's `y` rect minimum size value need to also be set to 342. Other than that I think this looks good.

**Comment by ender-titan1** [comment]
done

**Review by Kasterisk — APPROVED** [review]

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**